packages/workspace/src/node/default-workspace-server.ts

```diff
@@ -46,32 +46,31 @@ export class WorkspaceCliContribution implements CliContribution {
 @injectable()
 export class DefaultWorkspaceServer implements WorkspaceServer {
 
-    protected root: Promise<string>;
+    protected root: Promise<string | undefined>;
 
     constructor(
         @inject(WorkspaceCliContribution) protected readonly cliParams: WorkspaceCliContribution,
         @inject(ILogger) protected readonly logger: ILogger
     ) {
-        this.root = this.getRootURI();
+        this.root = this.retrieveRootURI();
```

Comment:
I find it really hard to tell if something is a uri as a string when it's not encoded in the type or in the value name.
I guess this is the reason we have functions called `getRootURI` or `retrieveRootURI`
Feels like the value should be thus called `rootURI` ?